includes/admin/class-amp-post-meta-box.php

```diff
@@ -213,6 +213,27 @@ public function enqueue_block_assets() {
 			return;
 		}
 
+		// Gutenberg v5.4 was bundled with WP 5.2, which is the earliest release known to work without errors.
+		$gb_supported = defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '5.4.0', '>=' );
+		$wp_supported = ! $gb_supported && version_compare( get_bloginfo( 'version' ), '5.2', '>=' );
+
+		// Let the user know that block editor functionality is not available if the current Gutenberg or WordPress version is not supported.
+		if ( ! $gb_supported && ! $wp_supported ) {
+			wp_add_inline_script(
```

Comment:
How about wrapping this in a `current_user_can( 'manage_options' )` check? It doesn't do much good to show this warning to non-admin users.

Reply:
Yea that's a good idea.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/5441/commits/7d9ea2babf8b815df48ef3a2bfe6c732226342a4.